### ISSUE TITLE
Customizable typeface for PDF preview or export

### ISSUE THREAD

**Issue by vekuda** [piersdeseilligny/betterfountain#18]
Personally I am fine with Courier Prime, but superlou's 'fountain' extension for Atom also allows to pick Anonymous Pro, Courier Code or GNU Typewriter... it could be a useful feature to have.

**Comment by piersdeseilligny** [comment]
I don't really want to embed a limited set of fonts into betterfountain, as it would be a lot more elegant to just use system fonts. However `pdfkit` doesn't currently support this (as it doesn't know where to look for the `.ttf` files associated with a font name). So this feature is currently on hold.

**Comment by piersdeseilligny** [comment]
![image](https://user-images.githubusercontent.com/42496508/56085179-47ad4880-5e3f-11e9-8a6a-e5d10c810fb5.png)

Looks like I talked too fast...

**Comment by vekuda** [comment]
Are you telling me that you managed to add this feature?

Also my congratulations (and many thanks) for having added auto-add CONT'D!

**Comment by piersdeseilligny** [comment]
@vekuda yup, it will be in 1.3.0. It's probably still slightly buggy, but it works.

And what do you mean auto-add CONT'D? I haven't added it yet.

**Comment by vekuda** [comment]
Ah, I had seen it in the milestone, but I didn't notice that it was still open.

So how is this typeface feature supposed to work? Will it use OS fonts or a bunch of built-in fonts?

**Comment by piersdeseilligny** [comment]
It will use os fonts, including autocomplete and an in-editor error if the font can't be found:

<img src="https://user-images.githubusercontent.com/42496508/56085542-57c82680-5e45-11e9-8909-b5dad6f1bda5.png" width="300">

I got the PDF export working by using a really cool npm module called [font-finder](https://github.com/princjef/font-finder) which allows me to point pdfkit to the correct files based simply off the name of the font.

**Comment by vekuda** [comment]
Wow, that's very clever!

I don't know how VS Code works, but this `font-finder` thing could be useful for an issue that I recently opened on VS Code git. 

Maybe not being a programmer I don't get along with terminals, but for me it's just too awkward that I can't preview in the settings the font that I want to use in the editor of VS Code. So that's exactly what I requested: I want a window to see the list of the OS fonts that I have on my machine and I want to be able to preview each typeface before deciding. 

The issue already existed... it is open since immemorable time, but nobody would work on it.
